tools/lint-hotkeys

```diff
@@ -104,7 +117,16 @@ def get_hotkeys_file_string() -> str:
         for help_text, key_combinations_list in categories[action]:
             various_key_combinations = " / ".join(
                 [
-                    " + ".join([f"<kbd>{key}</kbd>" for key in key_combination.split()])
+                    " + ".join(
+                        [
+                            f"<kbd>Shift</kbd> + <kbd>{key}</kbd>"
+                            if len(key) == 1 and key.isupper()
+                            else f"<kbd>{key.capitalize()}</kbd>"
+                            if len(key) == 1
+                            else f"<kbd>{key + DIRECTION_TO_SYMBOL_MAP.get(key, '')}</kbd>"
+                            for key in key_combination.split()
```

Comment:
I replied separately in the stream, so we may not need this change as it stands.

However, when inline conditionals get large they get more difficult to read, so at a minimum I'd go with something like the following:
```suggestion
                            f"<kbd>Shift</kbd> + <kbd>{key}</kbd>"
                            if len(key) == 1 and key.isupper()
                            else (
                                f"<kbd>{key.capitalize()}</kbd>"
                                if len(key) == 1
                                else f"<kbd>{key + DIRECTION_TO_SYMBOL_MAP.get(key, '')}</kbd>"
                            )
                            for key in key_combination.split()
```

However, while it requires rewriting the comprehensions/loops, it may well be clearer to simply express it as regular `if` statements instead - particularly given one of the inline conditionals is itself a ` + `-separated string.